docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -451,11 +453,11 @@ Each tweet has a unique ID and the comments and replies are tracked in "conversa
 per conversation. We can use these `conversation_id` values to get all comments and replies for a particular tweet.
 
 We use `conversation_id` value as the query to the `client.search_all()` function.
-If you want to see the particular conversation in the web browser, you can insert the tweet ID in the following URL: `https://twitter.com/anyuser/status/<tweet_id>`. For example, the first tweet from `df` dataframe has `tweet_id` of 1613280296316686337. By pasting the value into the URL, we get the following link - [https://twitter.com/anyuser/status/1613280296316686337](https://twitter.com/anyuser/status/1613280296316686337) that will show us the tweet, its comments and replies.
+If you want to see the particular conversation in the web browser, you can insert the tweet ID in the following URL: `https://twitter.com/anyuser/status/<tweet_id>`. For example, the first tweet from `df` dataframe has `tweet_id` of 1613280296316686337. By pasting the value into the URL, we get the following link - [https://twitter.com/anyuser/status/1613280296316686337](https://twitter.com/anyuser/status/1613280296316686337){:target="_blank"} that will show us the tweet, its comments and replies.
 
 The following function can be used to get all comments based on tweet's conversation ID:
 
-```python
+```bash linenums="1" hl_lines="8-8" title="python"
```

Comment:
python

Reply:
I think you're looking at outdated file but I changed all python to py